config/custom-environment-variables.yml

```diff
@@ -86,6 +86,7 @@ private:
   jenkins_pass: 'JENKINS_PASS'
   jira_user: 'JIRA_USER'
   jira_pass: 'JIRA_PASS'
+  librariesio_tokens: 'LIBRARIESIO_TOKENS'
```

Comment:
opted to just stick an `s` on the end to, along with the docs, convey this supports accepting multiple tokens. I'm not sure we have any prior art with this type of value beyond the various `*origins` vars though, so let me know if folks have any preferred alternatives